content/02.main-text.md

```diff
@@ -1,8 +1,21 @@
 ## Introduction {.page_break_before}
 
-Openness in research benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
-Improving the availability of code and data helps ensure the methods are transparent and the results can be reproduced  [@doi:10.1038/s41562-016-0021].
-Publishing in an open access journal increases the number of readers and reviewers [@doi:10.1371/journal.pone.0000308].
+The way science is performed is undergoing a terrific transition towards openness.
```

Comment:
This seems hard to quantify or directly support for a reference.  That has been my experience anecdotally, but is that representative of science in general?

Reply:
An indirect way of supporting this would be a plot of the number of papers published in open access journals as a proportion of all papers published (per year). Surely this has risen from near zero a few decades ago to something not-insignificant now. I'm not suggesting we do ourselves, but there may be such a graph floating around somewhere.

Reply:
> An indirect way of supporting this would be a plot of the number of papers published in open access journals as a proportion of all papers published (per year).

Our subsequent citation of https://doi.org/10.7717/peerj.4375 as well as several more... it's now well established that 30-50% of publications are OA.

However, I view the rise of OA as one small component of the "terrific transition". Code & data availability are at least as great. As is version control and electronic lab notebooks.

Reply:
Could we site policies towards openness. Like CZI, https://www.chanzuckerberg.com/science,
GDS, https://www.genome.gov/27562511/genomic-data-sharing-gds-policy/ ?